File: app/src/main/java/in/completecourse/UpdateVersionActivity.java
package in.completecourse;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.widget.Button;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.google.android.gms.ads.AdRequest;
import com.google.android.gms.ads.AdView;

public class UpdateVersionActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_update_version);

        //Use an activity context to get the rewarded video instance
        AdView mAdView = findViewById(R.id.adView);
        AdRequest adRequest = new AdRequest.Builder().build();
        mAdView.loadAd(adRequest);

        Button updateNowButton = findViewById(R.id.update_now_button);
        updateNowButton.setOnClickListener(v -> {
            Intent intent = new Intent("android.intent.action.VIEW", Uri.parse("https://play.google.com/store/apps/details?id=in.completecourse"));
            startActivity(intent);
        });
    }
}

File: app/src/main/java/in/completecourse/UpdateVersionActivity.kt
package `in`.completecourse

import `in`.completecourse.databinding.ActivityUpdateVersionBinding
import android.content.Intent
import android.net.Uri
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class UpdateVersionActivity : AppCompatActivity() {
    private lateinit var binding: ActivityUpdateVersionBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityUpdateVersionBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)

        binding.updateNowButton.setOnClickListener {
            val intent = Intent("android.intent.action.VIEW",
                    Uri.parse("https://play.google.com/store/apps/details?id=in.completecourse"))
            startActivity(intent)
        }
    }
}